model/model_training/requirements.txt

```diff
@@ -0,0 +1,19 @@
+bitsandbytes==0.36.0.post2
+datasets==2.8.0
+deepspeed==0.7.7
+evaluate==0.4.0
+fastlangid==1.0.11
+flash_attn==0.2.8
+gdown==4.7.1
+ipython==8.11.0
```

Comment:
hm. normally the requirements should reside in `pyproject.toml` which is now in the `model/` directory (parent of model_training). The file was moved allow to make model_training a "real" python module that can be imported everywhere.

Reply:
moved dependencies to pyproject.toml and deleted requirements.txt